design/authenticators/authenticator_management_improvement.md

```diff
@@ -0,0 +1,247 @@
+# Authenticators Management Improvement Proposal
+
+- [Authenticators Management Improvement Proposal](#authenticators-management-improvement-proposal)
+  - [Introduction](#introduction)
+  - [Proposed Solution](#proposed-solution)
+    - [Add Authenticator](#add-authenticator)
+      - [Functionality](#functionality)
+      - [Synopsis](#synopsis)
+      - [Example](#example)
+    - [Delete Authenticator](#delete-authenticator)
+      - [Functionality](#functionality-1)
+      - [Synopsis](#synopsis-1)
+      - [Example](#example-1)
+    - [Update Authenticator](#update-authenticator)
+      - [Functionality](#functionality-2)
+      - [Synopsis](#synopsis-2)
+      - [Example](#example-2)
+    - [Enable Authenticator](#enable-authenticator)
+      - [Functionality](#functionality-3)
+      - [Synopsis](#synopsis-3)
+      - [Example](#example-3)
+    - [Disable Authenticator](#disable-authenticator)
+      - [Functionality](#functionality-4)
+      - [Synopsis](#synopsis-4)
+      - [Example](#example-4)
+  - [Backwards Compatibility](#backwards-compatibility)
+  - [Performance](#performance)
+  - [Security](#security)
+  - [Documentation](#documentation)
+  - [Drawbacks](#drawbacks)
+  - [Implementation Options](#implementation-options)
+    - [Alternative 1 - Server Side](#alternative-1---server-side)
+      - [Pros](#pros)
+      - [Cons](#cons)
+    - [Alternative 2 - CLI Side](#alternative-2---cli-side)
+      - [Pros](#pros-1)
+      - [Cons](#cons-1)
+  - [Version Update](#version-update)
+  - [Delivery Plan](#delivery-plan)
+  
+## Introduction
+
+Conjur authenticators are defined as sub-policies, under the `conjur` policy.  
+Currently to define an authenticator, the user needs to complete the following steps:
+
+- Read the documentation
+- Understand how the authenticator policy is structured
+- Create a policy yaml file that defines the authenticator
+- Load the policy using Conjur CLI
```

Comment:
It's worth to mention an additional step for most of the authenticators - load values into variables created in the policy

Reply:
Right, I forgot to mention that step. I'll fix it. Thanks!